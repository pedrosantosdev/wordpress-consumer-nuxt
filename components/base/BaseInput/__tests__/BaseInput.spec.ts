import { render } from '@testing-library/vue'
import BaseInputVue from '@/components/base/BaseInput/BaseInput.vue'

describe('BaseInputVue', () => {
  it('should render properly', () => {
    const wrapper = render(BaseInputVue, {
      props: {
        label: 'Type Here',
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
