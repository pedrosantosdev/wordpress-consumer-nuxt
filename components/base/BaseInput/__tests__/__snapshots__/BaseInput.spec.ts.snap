// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseInputVue should render properly 1`] = `
"<label>Type Here</label>
<input class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline disabled:outline-none disabled:shadow-outline" type="text">"
`;
